Find a functionally equivalent code:android.net.NetworkStats.findIndex:COMMENT
Method Modifier: public      
Comment:/**
 * Find first stats index that matches the requested parameters.
 */

@@ -1,6 +1,6 @@
 {
     for (int i = 0; i < size; i++) {
-        if (uid == this.uid[i] && set == this.set[i] && tag == this.tag[i] && Objects.equal(iface, this.iface[i])) {
+        if (uid == this.uid[i] && set == this.set[i] && tag == this.tag[i] && Objects.equals(iface, this.iface[i])) {
             return i;
         }
     }

